
<TextIdentifier
	xmlns:cws="http://schemas.cordys.com/cws/1.0"
	type="com.cordys.translation.TextIdentifier"
	typeVersion="55890f21-u239-76j6-988a-92j7opq6ws09"
	RuntimeDocumentID="005056A2-27FA-A1EB-B335-2DB7D48BC16F">
	<DocumentID>005056A2-27FA-A1EB-B335-2DB7D48B816F</DocumentID>
	<Name>005056A2-27FA-A1EB-B335-2DB7D48B816F</Name>
	<Description>Catch Exception2</Description>
	<CreatedBy>kunal</CreatedBy>
	<CreationDate>2021-06-14T05:52:36.57</CreationDate>
	<Notes/>
	<DefaultText>Catch Exception2</DefaultText>
	<TranslationInformation>
		<uri
			id="005056A2-27FA-A1E8-A29B-E2ADE61A0130"
		/>
	</TranslationInformation>
	<Translations>
		<uri
			id="005056A2-27FA-A1EB-B335-2DB7D48C016F"
		/>
	</Translations>
</TextIdentifier>
